.evergreen/run-tests.sh

```diff
@@ -261,7 +261,7 @@ if [ -n "$COVERAGE" ] && [ "$PYTHON_IMPL" = "CPython" ]; then
     # coverage 7.3 dropped support for Python 3.7, keep in sync with combine-coverage.sh.
     # coverage >=5 is needed for relative_files=true.
     python -m pip install pytest-cov "coverage>=5,<7.3"
-    TEST_ARGS="$TEST_ARGS --cov pymongo --cov-branch --cov-report term-missing:skip-covered"
```

Comment:
These flags override the coverage settings in pyproject which we don't want to do.

Reply:
`--cov-report` just changes whats printed to the terminal. I removed it because the default behavior seems good enough.